applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,165 @@
 # Imaging AI Whole Body Segmentation
 
-This application uses MONAI re-trained TotalSegmentator model to segment 104 body parts from a DICOM series of a CT scan. It is implemented using Holohub DICOM processing operators and PyTorch inference operators.
+This application demonstrates the use of MONAI's TotalSegmentator model for whole-body segmentation of CT scans.
```

Comment:
The main purpose is not to demo the TotalSegmentator, rather the use of the medical imaging operators to build and package an app that parses medical images in DICOM and performs inference using A MONAI model.